### ISSUE TITLE
is there any way to do inference using intel inbuilt GPU? 

### ISSUE THREAD

**Issue by SahilChachere** [pjreddie/darknet#1779]
my system doesn't have an external GPU but possesses intel inbuilt UHD GPU 4GB, looking for a way to do inference using intel GPU? is there any way to do so?

**Comment by sowson** [comment]
Yes, you may use my OpenCL version fork at https://github.com/sowson/darknet Thanks!

**Comment by SahilChachere** [comment]
@sowson  How's the performance in terms of training and inference for yolov3 without GPU but using opencl??
